tests/function-controller/Dockerfile

```diff
@@ -15,14 +15,14 @@ RUN GO111MODULE=on go mod download
 
 COPY ./testsuite/ ${BASE_APP_DIR}/testsuite/
 COPY ./pkg/ ${BASE_APP_DIR}/pkg/
-COPY ./main_test.go ${BASE_APP_DIR}
+COPY ./cmd/ ${BASE_APP_DIR}/cmd/
 COPY ./licenses/ ${BASE_APP_DIR}/licenses/
 
 #
 # Build tests
 #
 
-RUN CGO_ENABLED=0 go test main_test.go -c
+RUN CGO_ENABLED=0 go build -o main.test ./cmd/main.go
```

Comment:
Go back to the proper way -> right now we're running test suite in that binary, so please compile it as test binary